### ISSUE TITLE
Please add an "disable antivirus warn" before applying settings

### ISSUE THREAD

**Issue by lucafont2** [Sophia-Community/SophiApp#28]
After applying settings my antivirus nuked the app

![immagine](https://user-images.githubusercontent.com/62725689/147822576-8998e99f-3313-42e2-9fb1-0379e86d11fa.png)

Consider adding a warn to disable antivirus also to avoid nuke before finishing the apply process

**Comment by lucafont2** [comment]
I dont know if it is related to what happened with the andivirus but now the program crashes every time after some settings are applied to the system

**Comment by farag2** [comment]
I'll chek all the issues in January after the New year eve. Time to take a rest before the beta 2. :)

**Comment by lucafont2** [comment]
> Don't think it's possible to fix. Such shitty AVs have unexpected behaviour. I advice to use a normal one, Defender.

Yes no problem, mine was a suggestion to add like a little popup to warn the user to disable AV before apply. 
Have a nice day

**Comment by farag2** [comment]
> mine was a suggestion to add like a little popup to warn the user to disable AV before apply. Have a nice day

I don't consider giving a popup for users to make them disabling their AV. This makes no sense). Better to use a normal one instead of such as yours. 🐱

**Comment by lucafont2** [comment]
> > mine was a suggestion to add like a little popup to warn the user to disable AV before apply. Have a nice day
> 
> I don't consider giving a popup for users to make them disabling their AV. This makes no sense). Better to use a normal one instead of such as yours. 🐱

Mine was a suggestion and i did not expected this be done. I just wanted to give a feedback to support the project, so I don't understand why beeing rude.
This program mass changes windows system settings so i think an antivirus seeing this as suspicious behaviour is not a " make no sense " thing.
It was better if you closed this ignoring.

**Comment by farag2** [comment]
I did not want to offend you in any kind. :)

> This program mass changes windows system settings so i think an antivirus seeing this as suspicious behaviour

AVs don't work in such way. Anyway, the suggestion is impossible to implement to. At least in the 1.0 version. We're collecting all suggestion to think them as an improvements for 2.0. have a good day! 🐱
